application_service/app_service.mak

```diff
@@ -62,82 +66,82 @@ clean:
 	@echo "removing executable file"
 	rm -rf $(EXE_DIR)/app_service.exe
 
-hello_world.exe: hello_world.cc
-	@echo "hello_world.cc"
-	$(CC) $(CFLAGS) -o $(O)/hello_world.exe $(S)/hello_world.cc
+hello_world.exe: $(S)/hello_world.cc
+	@echo "compiling $<"
+	$(CC) $(CFLAGS) -o $(@D)/$@ -c $<
```

Comment:
This is the standard pattern of change you will see ... on L70 and on L71. Just follow this pattern, to ease the review cycle.